utest/test_python_usage.py

```diff
@@ -76,3 +76,18 @@ def test_promise_handling(browser, application_server):
     browser.promise_to_upload_file(file.resolve())
     browser.click("#file_chooser")
     assert browser.get_text("#upload_result") == "test_python_usage.py"
+
+
+def test_wait_for_promise_to(browser):
```

Comment:
There should be also acceptance test, you can copy paste the example from the issue https://github.com/MarketSquare/robotframework-browser/issues/1685#issue-1105409364